crowbar_framework/app/models/service_object.rb

```diff
@@ -1551,26 +1544,6 @@ def run_remote_chef_client(node, command, logfile_name)
     }
   end
 
-  def chef_daemon(action, node_list)
```

Comment:
@aspiers then we should also remove `run_service` in the `node_object` as this is not used and `.service_command` in chef_object as this is not used either.